scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -261,6 +260,44 @@ workflow GvsQuickstartIntegration {
         }
     }
 
+    if (run_bge_integration) {
+        call QuickstartVcfIntegration.GvsQuickstartVcfIntegration as QuickstartVcfBgeIntegration {
+            input:
+                git_branch_or_tag = git_branch_or_tag,
+                git_hash = GetToolVersions.git_hash,
+                use_VQSR_lite = true,
+                extract_do_not_filter_override = false,
+                dataset_suffix = "bge",
+                use_default_dockers = use_default_dockers,
+                gatk_override = if (use_default_dockers) then none else BuildGATKJar.jar,
+                is_wgs = false,
+                interval_list = full_exome_interval_list,
+                expected_output_prefix = expected_output_prefix,
+                sample_id_column_name = sample_id_column_name,
+                vcf_files_column_name = vcf_files_column_name,
+                vcf_index_files_column_name = vcf_index_files_column_name,
+                sample_set_name = bge_sample_set_name,
+                drop_state = "FORTY",
```

Comment:
not for today! But it would be interesting to see if "Forty" is still the overwhelming majority of sites in BGEs though I suppose a tight interval list would prevent "Zero" from overshadowing it